core/iwasm/libraries/libc-wasi/sandboxed-system-primitives/src/posix.c

```diff
@@ -172,6 +172,38 @@ convert_errno(int error)
 #undef X
     return code;
 }
+#ifndef BH_PLATFORM_WINDOWS
+/*
+ * wasi_clockid_to_clockid is a plceholder in this file so that poll_oneoff complies,
+ * the implementation has been refactored to posix_clock.c. In the near future
+ * poll_oneoff will be refactored and the below placeholder will be removed.
+ */
+bool
+wasi_clockid_to_clockid(__wasi_clockid_t in, clockid_t* out) 
+    {
```

Comment:
Invalid coding style, could you use clang-format-12 to format the code?
https://github.com/bytecodealliance/wasm-micro-runtime/actions/runs/6573605221/job/17856981299